src/storage/memory_cache/write_back_task.rs

```diff
@@ -131,18 +179,56 @@ where
         self.lru_queue
             .retain_with_order(|&BlockCoordinate(i, _)| i != ino);
 
+        let mut handles = vec![];
+        let mut shutdown = self.block_flush_spawn_handle.is_shutdown();
+
         if let Some(file_level_pending_blocks) = file_level_pending_blocks {
-            for (block_id, (block, sender)) in file_level_pending_blocks {
-                let res = self.storage.backend().store(ino, block_id, block).await;
-                sender.send(res).unwrap_or_else(|res| {
-                    error!("The receiver is closed unexpectedly, with storage result: {res:?}");
-                });
+            for (block_id, (block, tx)) in file_level_pending_blocks {
+                if shutdown {
```

Comment:
When a user initiates a logout, I believe we only need to ensure the program exits normally. Try not to do anything extra as much as possible